### ISSUE TITLE
TypeError: smart_charging() missing 1 required positional argument: 'enable'

### ISSUE THREAD

**Issue by alexanderwink** [fondberg/easee_hass#124]
Why I'm i getting this error? I know there was a previous issue reported on this but since then pyeasee and easee_hass has been updated.

From HA log:

`INFO (MainThread) [pyeasee.easee] Easee python library version: 0.7.36`

From manifest.json:

```
"name": "Easee EV charger",
"version": "0.9.31",
```

**Comment by olalid** [comment]
Hi, I am missing a bit of context here. Under what circumstances are you getting this error? Are you doing anything specific like calling a service?

**Comment by tmjo** [comment]
When calling the service smart_charging which is supposed to toggle the smart charging on/off. I get the same error, seems like we (possibly me) must have forgotten to push this change to the HA integration when it was fixed in pyeasee [PR](https://github.com/fondberg/pyeasee/pull/40), or something has happened since. Anyway, I can fix this when I find the time.

**Comment by alexanderwink** [comment]
Yes @tmjo is correct. This happens when I call the smart_charging service from HA/nodered.
